src/Rector/v8/v6/RefactorTCARector.php

```diff
@@ -167,21 +167,25 @@ private function addFieldControlInsteadOfWizardsAddListEdit(Array_ $items): void
                 continue;
             }
 
-            foreach ($fieldValue->value->items as $configValue) {
-                if (null === $configValue) {
+            $fieldValueArray = $fieldValue->value;
+            foreach ($fieldValueArray->items as $configValue) {
+                if (! $configValue instanceof ArrayItem) {
                     continue;
                 }
 
                 if (! $configValue->value instanceof Array_) {
                     continue;
                 }
 
+                /** @var Array_ $configValueArray */
+                $configValueArray = $configValue->value;
```

Comment:
This is sometimes useful, because PHPStan/IDE can lose type of single nested variables.